README.md

```diff
@@ -71,9 +72,31 @@ _Figure 2: ODE UI demonstrating message subscription_
 
 Notice that the empty fields in the J2735 message are represented by a `null` value. Also note that ODE output strips the MessageFrame header and returns a pure BSM or TIM in the subscription topic.
 
-
 With the PPM module running, all filtered BSMs that are uploaded through the web interface will be captured and processed. You will see an output of both submitted BSM and processed data unless the entire record was filtered out.
 
+<b>For testing decoding with all supported ODE message types:</b>
+
+To test decoding all supported ODE messages, the UDP endpoints must be utilized. These endpoints specifically take hex ASN1 encoded message data. These messages are allowed to have headers but do not need to.
```

Comment:
Is this hex ASN.1 encoding UPER encoded? or?

Reply:
What kind of headers?

Reply:
Whichever one the ASN.1 library supports. Not actually certain. The headers are trimmed out at the moment so the type of header isn't super important. How would you like me to specify this?

Reply:
@dan-du-car I have gone ahead and made adjustments to specify these.

Reply:
Thank you!